ui/raidboss/data/04-sb/dungeon/temple_of_the_fist.js

```diff
@@ -50,5 +121,202 @@
         fr: 'trouvez une zone safe',
       },
     },
+    {
+      id: 'Temple Spirit Wave',
+      regex: / 14:1FE7:Ivon Coeurlfist starts using Spirit Wave/,
+      condition: function(data) {
+        return data.role == 'healer';
+      },
+      infoText: {
+        en: 'AoE',
+      },
+    },
+    {
+      id: 'Temple Touch Of Slaughter',
+      regex: / 14:1FE6:Ivon Coeurlfist starts using Touch Of Slaughter on (\y{Name})/,
+      alertText: function(data, matches) {
+        if (data.me == matches[1]) {
+          return {
+            en: 'Slaughter on YOU',
+          };
+        } else if (data.role == 'healer') {
+          return {
+            en: 'Heal ' + data.shortName(matches[1]) + ' soon',
+          };
+        }
+      },
+    },
+    {
+      id: 'Temple Coeurl Heads',
+      regex: / 03:\y{ObjectId}:Added new combatant Head Of The Coeurl/,
+      suppressSeconds: 5,
+      infoText: function(data) {
+        // Ivon summons invisible heads when he first appears. This avoids notifying.
+        if (data.firstHeads) {
+          return {
+            en: 'Avoid floating heads',
+          };
+        }
+      },
+      run: function(data) {
+        data.firstHeads = data.firstHeads || true;
+      },
```

Comment:
Does `data` get cleared upon wipe? This might have interesting side effects if not.

Reply:
That's a good point, no, it doesn't. But, checking back through my logs, there's an instant cast that Ivon uses that *should* match this timing closely enough, so it might be possible to just avoid doing this altogether.

Reply:
Yes, this definitely is unnecessary. Coeurl Whisper is used invisibly to summon the heads, but I didn't even think to use it because it's instant-cast without much animation. Thanks for catching that!

Reply:
Update: Data is cleared upon re-entering combat (as defined by the game's definition, not ACT's), see [oopsyraidsy's README](https://github.com/quisquous/cactbot/blob/master/ui/oopsyraidsy/data/README.md#data-fields).

Reply:
Well then. Good to know that it *would* have worked, but it's just as well we went with something else in this situation, alerting based on ability use is *probably* better than alerting on added combatants.